.cspell.json

```diff
@@ -164,7 +164,8 @@
     "WBITS",
     "laddr",
     "nrows",
-    "Dumpable"
+    "Dumpable",
+    "aiter"
```

Comment:
typo?

Reply:
I use response.aiter_lines(), and spell check will raise error: unknown word "aiter"